src/utils/beneficial.owner.other.ts

```diff
@@ -129,6 +142,13 @@ const setBeneficialOwnerData = (reqBody: any, id: string): ApplicationDataType =
     : {};
   data[StartDateKey] = mapFieldsToDataObject(reqBody, StartDateKeys, InputDateKeys);
 
+  // not present in register journey
+  if ("is_still_bo" in reqBody) {
+    data[CeasedDateKey] = (reqBody["is_still_bo"] === '0')
```

Comment:
also candidate for shared method

Reply:
in my PR I've just done the === '0' as anything else means not ceased. So no need for common methods as just one line.